Python/twitter-automation/README.md

```diff
@@ -0,0 +1,34 @@
+# Twitter automation
+- - - - - - - - 
+This is a Twitter script that can automatically like and retweet tweets containing user-specified keywords.</br>
+This bot uses the Tweepy stream to actively watch for tweets that contain certain keywords. For each tweet, if you’re not the tweet author, it will mark the tweet as Liked and then retweet it.
+
+## Requirements
+```pip install tweepy```</br>
+
+## Setting up the twitter API
```

Comment:
We are not able to follow up with the instructions you have written in the readme. please check again the links and steps you have mentioned. If possible please add an image to clarify things.

Reply:
Sure,
I'll do that